contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,172 @@
+---
+date: 2023-04-27
+title: "8 A/B testing mistakes and how to avoid them"
```

Comment:
Trying to think of ways to punch up this headline. Some ideas:

- 8 dangerous A/B testing mistakes (and how to avoid them)
- 8 A/B testing mistakes every engineer should know
- 8 annoying A/B testing mistakes (and how to avoid them)
- 8 costly A/B testing mistakes (and how to avoid them)

I like the last couple – "dangerous" is too. The 2nd isn't too bad, either. Thoughts?

Reply:
Agree the punchline is boring. I like the "every engineer should know"

How about: "8 annoying A/B testing mistakes every engineer should know"